bitcoin/src/blockdata/block.rs

```diff
@@ -68,90 +66,32 @@ impl BlockHeader {
     }
 
     /// Computes the target [0, T] that a blockhash must land in to be valid.
```

Comment:
OT: This looks like an array, got me confused.

Reply:
Good point, fixed.